test/nni_test/nnitest/validators.py

```diff
@@ -33,6 +33,14 @@ def __call__(self, rest_endpoint, experiment_dir, nni_source_dir, **kwargs):
             print('\n\n')
         remove('report.json')
 
+class ImportValidator(ITValidator):
+    def __call__(self, rest_endpoint, experiment_dir, nni_source_dir, **kwargs):
+        exp_id = osp.split(experiment_dir)[-1]
+        import_data_file_path = kwargs.get('import_data_file_path')
+        proc = subprocess.run(['nnictl', 'experiment', 'import', exp_id, '-f', import_data_file_path])
+        assert proc.returncode == 0, \
+            '`nnictl experiment import {0} -f {1}` failed with code {2}'.format(exp_id, import_data_file_path, proc.returncode)
```

Comment:
Any other means to verify the data is actually imported successfully besides checking the return code?

Reply:
I'll view the code again to find if there is a way to check the result.

Reply:
In order to check the imported data, add an api `api/v1/nni/experiment/imported-data` to get the imported data. I don't know if it is appropriate to do so. Maybe users also have demand to view the data they have imported?

Reply:
It is OK to call `api/v1/nni/experiment/imported-data` to verify